src/bench/wallet_balance.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2012-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <bench/bench.h>
+#include <interfaces/chain.h>
+#include <key_io.h>
+#include <optional.h>
+#include <test/util.h>
+#include <validationinterface.h>
+#include <wallet/wallet.h>
+
+struct WalletTestingSetup {
+    std::unique_ptr<interfaces::Chain> m_chain = interfaces::MakeChain();
+    CWallet m_wallet;
+
+    WalletTestingSetup()
+        : m_wallet{m_chain.get(), WalletLocation(), WalletDatabase::CreateMock()}
```

Comment:
Could get rid of WalletTestingSetup constructor and just initialize m_wallet directly above.